tests/test_utils.py

```diff
@@ -5,4 +5,4 @@
 
 def test_parse_time():
     great_date = datetime(1985, 8, 14, 7, 10, 1, 123456)
-    assert parse_time(great_date) == '14/08/1985, 07:10:01.123'
+    assert parse_time(great_date) == '14/08/1985 07:10:01.123'
```

Comment:
why did we erase the `,`?

Reply:
We should use a standard format specifier. https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings#GeneralDateLongTime
> The "G" standard format specifier represents a combination of the short date ("d") and long time ("T") patterns, separated by a space